src/applications/accredited-representative-portal/components/common/Header/MobileHeader/MobileLogoRow.jsx

```diff
@@ -39,9 +39,11 @@ export const MobileLogoRow = () => {
             type="button"
           >
             Menu
-            <i
+            <va-icon
+              size={4}
+              icon="see name mappings here https://design.va.gov/foundation/icons"
```

Comment:
the correct icon prop needs to be provided here or no icon will render.

There are several more of these issues in this PR as well that will need to be addressed in a similar fashion. I would say it would be better to revert these icon changes and allow the respective teams to update their individual icon usages.

Reply:
OH, I was so stuck on looking at the test that I didn't realize these changes had been pulled in somehow. I did a fairly large merge from main and must have pulled something in from someone else's work. I'll change these back on my local and push up for sure :D

Reply:
No worries. I have seen this actually happen once before as well. There was something with the eslint rule that automatically was 'fixing'/breaking these icons a while back, so that might be where the issue came from. Your newest commit looks good now. Thanks!